content/spin/v2/key-value-store-tutorial.md

```diff
@@ -298,22 +293,29 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
   switch (request.method) {
     case "POST":
       store.set(request.uri, request.body || (new Uint8Array()).buffer)
+      console.log(`Storing value in the KV store with ${request.uri} as the key`);
       break;
     case "GET":
       let val
       try {
         val = store.get(request.uri)
         body = decoder.decode(val)
+      	console.log(`Found value for the key ${request.uri}`);
       } catch (error) {
+      	console.log(`Key ${request.uri} not found`);
```

Comment:
I believe JS KV returns null for key not found rather than throwing (so this will need to be checked in the try block).  If I'm wrong we probably need to update our API guide

Reply:
Fixed